app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -13,9 +14,7 @@ import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.navigation.Navigation.findNavController
 import com.google.android.material.snackbar.Snackbar
-import kotlinx.android.synthetic.main.fragment_login.email
-import kotlinx.android.synthetic.main.fragment_login.loginButton
-import kotlinx.android.synthetic.main.fragment_login.password
+import kotlinx.android.synthetic.main.fragment_login.*
```

Comment:
Why marking as resolved without commit?

Reply:
@kshitijsuri90 is everything correct except that wildcard import.
Will change that wildcard import and will update the commit with changes.

Reply:
Cool. No need to resolve it now. It will automatically be resolved when you make the changed commit.

Reply:
Okay thanks @kshitijsuri90